tests/results/result_actions/IgnoreResultActionTest.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+
+from coalib.misc.ContextManagers import make_temp
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+from coalib.results.result_actions.IgnoreResultAction import IgnoreResultAction
+from coalib.settings.Section import Section
+
+
+class IgnoreResultActionTest(unittest.TestCase):
+
+    def test_apply(self):
+        uut = IgnoreResultAction()
+        with make_temp() as f_a, make_temp() as f_b, make_temp() as f_c:
+
+            file_dict = {
+                f_a: ["1\n", "2\n", "3\n"],
+                f_b: ["1\n", "2\n", "3\n"],
+                f_c: ["1\n", "2\n", "3\n"]
+            }
+            expected_file_dict = {
+                f_a: ["1\n", "2\n", "3\n//"],
+                f_b: ["1\n", "2\n", "3\n//"],
+                f_c: ["1\n", "2\n", "3\n//"]
+            }
+
+            file_diff_dict = {}
+
+            diff = Diff(file_dict[f_a])
+            uut.apply_from_section(Result("origin", "msg", diffs={f_a: diff}),
+                                   file_dict,
+                                   file_diff_dict,
+                                   Section("t"))
+
+            diff = Diff(file_dict[f_a])
+            diff.change_line(3, "3\n", "3\n//")
+            uut.apply_from_section(Result("origin", "msg", diffs={f_a: diff}),
+                                   file_dict,
+                                   file_diff_dict,
+                                   Section("t"))
+
+            diff = Diff(file_dict[f_b])
+            diff.change_line(3, "3\n", "3\n//")
+            uut.apply(Result("origin", "msg", diffs={f_b: diff}),
+                      file_dict,
+                      file_diff_dict)
+
+            for filename in file_diff_dict:
+                file_dict[filename] = file_diff_dict[filename].modified
+
+            self.assertEqual(file_dict, expected_file_dict)
+            with open(f_a) as fa:
+                self.assertEqual(file_dict[f_a], fa.readlines())
+            with open(f_b) as fb:
+                self.assertEqual(file_dict[f_b], fb.readlines())
+            with open(f_c) as fc:
+                # File c is unchanged and should be untouched
+                self.assertEqual([], fc.readlines())
```

Comment:
this looks utterly complicated, copied from ApplyPatchACtion? There's no need to do it as complicated as the applicpatchactiontest.